File: src/main/java/org/dukcode/ps/codetree/trail2/IntroRepeatedOutput.java
package org.dukcode.ps.codetree.trail2;

import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

/**
 * @see <a
 * href="https://www.codetree.ai/ko/trails/complete/curated-cards/intro-repeated-output/introduction">LINK</a>
 */
public class IntroRepeatedOutput {

  private static final String TEXT = "12345^&*()_";

  private static BufferedReader br;
  private static BufferedWriter bw;

  private static int n;

  public static void main(String[] args) throws IOException {
    br = new BufferedReader(new InputStreamReader(System.in));
    bw = new BufferedWriter(new OutputStreamWriter(System.out));

    n = Integer.parseInt(br.readLine());
    printLine(n);

    br.close();
    bw.close();

  }

  private static void printLine(int times) throws IOException {
    for (int i = 0; i < times; i++) {
      bw.write(TEXT);
      bw.newLine();
    }
  }
}

File: src/main/kotlin/org/dukcode/ps/codetree/trail02/chapter01/lesson01/IntroRepeatedOutput.kt
package org.dukcode.ps.codetree.trail02.chapter01.lesson01

import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.util.StringTokenizer

private val br = BufferedReader(InputStreamReader(System.`in`))
private val bw = BufferedWriter(OutputStreamWriter(System.out))
private var st: StringTokenizer? = null

/**
 * @see [LINK](https://www.codetree.ai/ko/trails/complete/curated-cards/intro-repeated-output/description)
 */
fun main() {
    val cnt = br.readLine().toInt()

    printLine(cnt)

    bw.flush()
}

fun printLine(cnt: Int) {
    repeat(cnt) {
        bw.write("12345^&*()_")
        bw.newLine()
    }
}
